docs/getting_started.md

```diff
@@ -26,6 +26,7 @@ Port | Protocol | Component | Function
 6832 | UDP      | agent     | accept jaeger.thrift over binary thrift protocol
 5778 | HTTP     | agent     | serve configs
 16686| HTTP     | query     | serve frontend
+14268 | TCP     | collector | TCP collector endpoint
```

Comment:
Can you please change it to HTTP?

Reply:
Sure, no problem

Reply:
Sorry, I am looking at other tables could you please add this to the last column  `accept jaeger.thrift directly from clients` or something similar encorporating jaeger.thrift :)

Reply:
Sure thing!

Reply:
PR rebased and includes the `jaeger.thrift` :)

Reply:
lol it says that `zipkin.thrift`

Reply:
Ouch... Stupid mistake :P. Sorry! Will fix ASAP

Reply:
Should be correct this time, unless the fat fingers once again got the best of me :)